backend/internal/auth/mock.go

```diff
@@ -0,0 +1,17 @@
+package auth
+
+import (
+	"context"
+	"strings"
+)
+
+type MockAuthenticator struct{}
+
+// This is an authenticator that takes tokens of the form test-token<user ID>
+func (a MockAuthenticator) VerifyToken(ctx context.Context, token string) (string, bool) {
+
+	if strings.HasPrefix(token, "test-token-") {
+		return strings.TrimPrefix(token, "test-token-"), true
```

Comment:
Incredibly minor but this could be modified so that the structure is
if (problem)
      return false

return true

In the same way that the Firebase VerifyToken is written. Most of our other code also has the structure of checking for problems in if statements, and only returning true at the end